_episodes/01-intro.md

```diff
@@ -73,6 +73,13 @@ weight_kg_text = 'weight in kilograms:'
 ~~~
 {: .language-python}
 
+To convert the weight from float to an integer we use the int function, the same way we use the print statement, as follows:
+
```

Comment:
@maxim-belkin what do you think about this proposed introduction of type conversion here? I know we are trying to keep from introducing too much new content that does not directly affect the continuation of the lesson due to time constraints and too much info for a workshop. This is a rather small addition though.

If we do want to include it, we should have backticks around `float` and `int`. Also, this is just before we mention `print`, so I think it would make sense to leave that reference out. Since this would  be the first mention of "function", we could link to the reference at https://swcarpentry.github.io/python-novice-inflammation/reference/#function.

Reply:
I would not introduce type conversion at this stage. In this episode, we mostly focus on learning about variables and how to define/print them. I don't find it necessary to discuss about type conversion.

Reply:
I concur with Anne -- I don't see how we can plug type conversion into this episode (it's too early and there is no real need for that here). We could do that in the episode where we deal with command line arguments if we needed to convert user input to `int` or `float`.